tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTestHelper.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.Framework.Docker.Tests
+{
+    public class ImageTestHelper
+    {
+        private static Lazy<JArray> ImageInfoData;
+
+        public DockerHelper DockerHelper { get; }
+        private ITestOutputHelper _outputHelper;
+
+        public ImageTestHelper(ITestOutputHelper outputHelper)
+        {
+            _outputHelper = outputHelper;
+            DockerHelper = new DockerHelper(outputHelper);
+        }
+
+        static ImageTestHelper()
+        {
+            ImageInfoData = new Lazy<JArray>(() =>
+            {
+                string imageInfoPath = Environment.GetEnvironmentVariable("IMAGE_INFO_PATH");
+                if (!String.IsNullOrEmpty(imageInfoPath))
+                {
+                    string imageInfoContents = File.ReadAllText(imageInfoPath);
+                    return JsonConvert.DeserializeObject<JArray>(imageInfoContents);
+                }
+
+                return null;
+            });
+        }
+
+        public static IEnumerable<object[]> GetVerifyImagesData(IEnumerable<ImageDescriptor> imageDescriptors)
```

Comment:
I feel like a more descriptive name would be helpful given this refactoring. `FilterImageData`, `ApplyImageDataFilters`, or `GetFilteredImageData`?

Reply:
I went with `ApplyImageDataFilters`.